proto/beacon/p2p/v1/types.proto

```diff
@@ -189,8 +189,8 @@ message ProposalSignedData {
 }
 
 message SlashableVoteData {
-  repeated ethereum.common.Uint32List aggregate_signature_poc_0_indices = 1; // proof of custody indices for 0 bits.
-  repeated ethereum.common.Uint32List aggregate_signature_poc_1_indices = 2; // proof of custody indices for 1 bits.
+  repeated uint32 aggregate_signature_poc_0_indices = 1; // proof of custody indices for 0 bits.
+  repeated uint32 aggregate_signature_poc_1_indices = 2; // proof of custody indices for 1 bits.
```

Comment:
This used to be a [[uint32]] but that was incorrect - the spec defines it as a [uint24]